### DISCUSSION TITLE
c# Recive UDP steam whit Emgu.CV.runtime.windows  4.5.3.4721

### DISCUSSION THREAD

**Discussion by barabba2005** [emgucv/emgucv#627] [Q&A]
Hi

I'm triyng to recive a UDP stream from IP camera.

If i use this line:
capture = new Emgu.CV.VideoCapture(@"D:\3.mxf");
i see my video file corretcly.

If i use this line:
capture = new Emgu.CV.VideoCapture("udp://127.0.0.1:2001");
i can't see anything

This is my code:

if (capture == null)
{
 capture = new Emgu.CV.VideoCapture(@"D:\3.mxf");
capture.ImageGrabbed += CaptureImmageGrabber;                
capture.Start();               
  }

private void CaptureImmageGrabber(object sender, EventArgs e)
        {
            textBox1.AppendText("xx");
            try
            {
                Mat m = new Mat();
                capture.Retrieve(m ,0);
                pictureBox1.Image = m.ToImage<Bgr, byte>().ToBitmap();
                Thread.Sleep(5);
            }
            catch (Exception ex)
            {              
            }           
        }

Thanks in advance for help

**Comment by barabba2005** [comment]
can someone help me?

Thanks

**Comment by emgucv** [comment]
This may be useful:
https://stackoverflow.com/questions/50916903/how-to-process-vlc-udp-stream-over-opencv

When creating the videocapture, try to use FFMPEG backend and see if that helps.

Please also make sure the udp stream can be viewed from other apps before trying to load it with Open CV / Emgu CV.

**Comment by barabba2005** [comment]
Thank for help.
I rewrote the FFmpeg script and now Emgu works fine.

ffmpeg -y -f dshow -r 25 -video_size 1920x1080 -pixel_format uyvy422 -i video="Decklink Video Capture" -s 640x480 -c:v libxvid  -r 25 -an  -f mpegts "udp://127.0.0.1:2001"

**Comment by emgucv** [comment]
Good to know.

Thank you for the update.

**Comment by metolina** [comment]
How can i fisheye return to this example https://stackoverflow.com/questions/49770563/virtual-ptz-camera-via-physical-fisheye-camera-and-open-cv
